http/responseemitter.go

```diff
@@ -175,11 +184,8 @@ func (re *responseEmitter) Flush() {
 }
 
 func (re *responseEmitter) preamble(value interface{}) {
-	h := re.w.Header()
-	// Expose our agent to allow identification
-	h.Set("Server", "go-ipfs/"+config.CurrentVersionNumber)
```

Comment:
Any reason to get rid of this?

Reply:
We now do this by adding it to the Headers in the ServerConfig. Since this lib isnt in go-ipfs (and shouldnt depend on it anymore) we cant import `config`